package.json

```diff
@@ -76,7 +76,7 @@
     "tslint-config-prettier": "^1.17.0",
     "tslint-config-security": "^1.16.0",
     "tslint-config-swimlane": "^5.0.0",
-    "typescript": "^4.3.5"
```

Comment:
why roll back TS?

Reply:
➜  ngx-datatable git:(chore/update-node-npm) ✗ npm install
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@"~4.2.3 || ~4.3.2" from @angular-devkit/build-angular@12.1.4
npm WARN node_modules/@angular-devkit/build-angular
npm WARN   dev @angular-devkit/build-angular@"~12.1.4" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@">=4.2.3 <4.4" from @angular/compiler-cli@12.1.5
npm WARN node_modules/@angular/compiler-cli
npm WARN   dev @angular/compiler-cli@"~12.1.4" from the root project
npm WARN   3 more (@angular-devkit/build-angular, @ngtools/webpack, ng-packagr)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@">=4.2.3 <4.4" from @angular/compiler-cli@12.1.5
npm WARN node_modules/@angular/compiler-cli
npm WARN   dev @angular/compiler-cli@"~12.1.4" from the root project
npm WARN   3 more (@angular-devkit/build-angular, @ngtools/webpack, ng-packagr)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@">=4.2.3 <4.4" from @angular/compiler-cli@12.1.5
npm WARN node_modules/@angular/compiler-cli
npm WARN   dev @angular/compiler-cli@"~12.1.4" from the root project
npm WARN   3 more (@angular-devkit/build-angular, @ngtools/webpack, ng-packagr)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@"~4.2.3 || ~4.3.2" from ng-packagr@12.2.5
npm WARN node_modules/ng-packagr
npm WARN   dev ng-packagr@"^12.1.2" from the root project
npm WARN   1 more (@angular-devkit/build-angular)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@"~4.2.3 || ~4.3.2" from ng-packagr@12.2.5
npm WARN node_modules/ng-packagr
npm WARN   dev ng-packagr@"^12.1.2" from the root project
npm WARN   1 more (@angular-devkit/build-angular)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@"^2.9.1 || ^3.0.1" from tslint-config-security@1.16.0
npm WARN node_modules/tslint-config-security
npm WARN   dev tslint-config-security@"^1.16.0" from the root project
npm WARN   1 more (tslint-config-swimlane)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer typescript@"^2.9.1 || ^3.0.1" from tslint-config-security@1.16.0
npm WARN node_modules/tslint-config-security
npm WARN   dev tslint-config-security@"^1.16.0" from the root project
npm WARN   1 more (tslint-config-swimlane)
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-datatable@0.0.0
npm WARN Found: typescript@4.2.4
npm WARN node_modules/typescript
npm WARN   dev typescript@"^4.3.5" from the root project
npm WARN   6 more (@angular-devkit/build-angular, ...)